.gitignore

```diff
@@ -1,12 +1,6 @@
 rebar3.crashdump
 .rebar3
 _*
-.eunit
-*.o
-*.beam
-*.plt
 erl_crash.dump
-rel/
 _build
-_checkouts
```

Comment:
Can we keep this one, to avoid experiments from being pushed?

Reply:
I mean, sure, my thinking was quite the opposite, though: if you have `_checkouts` and you don't see it as something to add to Git stage you think stuff's working when it's not, which is why I prefer to remove it (from `.gitignore`), but maybe there's reasons to keep it. I'll keep it, for the moment, as it seems to be your preference.

Mind you, I usually keep in `.gitignore` stuff that's "generated" and `_checkouts` is not "generated" but purposefully created by the consumer.

Reply:
0579fb2

Reply:
I tend to fail on the side of _using `git add .; git commit -am "…"` without looking at `git status` first_ much more regularly than on the side of _forgetting to push the corresponding changes to the checked out repo together with the changes to this one_. 🤷🏻

Reply:
Sure, I don't mind: if that's what most people do (I have no stat.s on this to support it) your reasoning is good; otherwise mine is too 😄

I do look at the stage closely first, though, but again I don't know how many people do that.